src/lammpsio/topology.py

```diff
@@ -0,0 +1,171 @@
+import numpy
+
+
+class Topology:
```

Comment:
Add some class level documentation, and document the constructor. You will also need an explicit `Attributes` section to document `num_types`.